cli/tests/e2e/e2e.nonQv.test.ts

```diff
@@ -71,23 +73,22 @@ describe("e2e tests", function test() {
 
   // before all tests we deploy the vk registry contract and set the verifying keys
   before(async () => {
+    signer = await getDefaultSigner();
+
     // we deploy the vk registry contract
-    await deployVkRegistryContract({});
+    await deployVkRegistryContract({ signer });
     // we set the verifying keys
-    await setVerifyingKeys(setVerifyingKeysNonQvArgs);
+    await setVerifyingKeys({ ...setVerifyingKeysNonQvArgs, signer });
   });
 
   describe("1 signup, 1 message (with signer as argument)", () => {
```

Comment:
let's remove (with signer as argument) as now it's mandatory so not specific to this test

Reply:
Will do it in #1190